src/settings-ui/Settings.UI/FlyoutWindow.xaml

```diff
@@ -17,17 +17,10 @@
     IsShownInSwitchers="False"
     IsTitleBarVisible="False"
     mc:Ignorable="d">
-    <winuiex:WindowEx.Backdrop>
-        <winuiex:AcrylicSystemBackdrop
-            DarkFallbackColor="#1c1c1c"
-            DarkLuminosityOpacity="0.96"
-            DarkTintColor="#202020"
-            DarkTintOpacity="0.5"
-            LightFallbackColor="#EEEEEE"
-            LightLuminosityOpacity="0.90"
-            LightTintColor="#F3F3F3"
-            LightTintOpacity="0" />
-    </winuiex:WindowEx.Backdrop>
+    <winuiex:WindowEx.SystemBackdrop>
+        <DesktopAcrylicBackdrop />
+    </winuiex:WindowEx.SystemBackdrop>
+    
     <Grid>
         <!-- HACK: https://github.com/microsoft/microsoft-ui-xaml/issues/7629 -->
```

Comment:
When this gets fixed, removing the entire grid and only leaving `<flyout:ShellPage x:Name="FlyoutsShellPage" />` keeps the transparent acrylic effect. Adding the grid here as a hack is adding another layer that blocks the acrylic.

Reply:
In other words: less is more 🙃